### PR TITLE
Adds waitedClicked for E2E tests

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

**Comment by rudolfix** [comment]
I really do not think that wait before click is really a problem

**Comment by Moejoe90** [comment]
@rudolfix It is a problem in some cases for example 
`Access Wallet Modal` 
I think most of these issues come one way or another with modals

File: app/test-e2e/utils/index.ts [hunk 1]
```diff
@@ -102,18 +102,18 @@ export const typeLightwalletRecoveryPhrase = (words: string[]) => {
...
-  cy.get(tid("btn-send")).click();
+  cy.get(tid("btn-send")).awaitedClick();
 };
 
 export const confirmAccessModal = (password: string) => {
   cy.get(tid("access-light-wallet-password-input")).type(password);
   cy.get(tid("access-light-wallet-confirm"))
-    .wait(1500)
-    .click();
+    .wait(1000)
```
Comment (desfero) [inline app/test-e2e/utils/index.ts]: hm, can we remove `wait` here and add param `timeToWait = 1000` so we can hide all `wait`s inside `.awaitedClick`? 
```
.awaitedClick(2000);
```

what do you think?

Comment (Moejoe90) [reply app/test-e2e/utils/index.ts]: good idea

**Pull request merged**